RESOURCES.md

```diff
@@ -0,0 +1,86 @@
+> Otherwise, you can assume a TRL-4 maturity.
+
+> [!WARNING]  
+> The CVA6 team is not liable for the other repositories.
+> Assess their content and make sure they fit your needs and are mature enough for design.
+> Plese direct your issues or pull requests to these external repositories.
+
+## Our legacy
+
+CVA6 was designed by the [PULP Platform team](https://www.pulp-platform.org/). You can integrate it with many other PULP designs from [github.com/pulp-platform](https://github.com/pulp-platform).
+
+## Technical resources
+
+### SW Tools and OSes
+
+RISC-V tools for CVA6 and Buildroot Linux support are available [here](https://github.com/openhwgroup/cva6-sdk).
+
+Yocto Linux support for CVA6 is available [here](https://github.com/openhwgroup/meta-cva6-yocto).
+
+FreeRTOS support for CVA6 is available [here](https://github.com/FreeRTOS/FreeRTOS-Partner-Supported-Demos/tree/main/RISC-V_cva6).
+
+Zephyr support for CV64A6 will soon be available.
+
+This [tutorial](https://github.com/ThalesGroup/cva6-eclipse-demo) offers resources to debug CVA6 under Eclipse IDE.
+
+The OS ports below are on Digilent Genesys 2 board.
+
+### Related building blocks
+
+These building blocks fit very nicely with CVA6:
+
+- [OpenPiton](https://github.com/PrincetonUniversity/openpiton) is a many-core framework that supports CVA6.
+- [Culsans/CV-TCCC](https://github.com/pulp-platform/culsans) is a multi-core infrastructure for a few CVA6 cores.
+- [ARA/CV-VEC](https://github.com/pulp-platform/ara) is a vector unit for CVA6.
+
+### Design examples (FPGA)
+
+The CVA6 repository contains the CVA6 core and a basic CPU design, the "APU" and its implementation on a Digilent Genesys 2 FPGA board. Here is a list of other CVA6-based FPGA designs:
+
+The [technical kits](https://github.com/thalesgroup/cva6-softcore-contest) of a student contest organized in France can be used as educational resources or as an easy way to get CVA6 up and running with a cheaper Digilent Zybo Z7-20 board. You will find in it:
+- The 2020-2021 contest, focusing on PPA optimization;
+- The 2021-2022 contest, focusing on energy optimization;
+- The 2022-2023 contest, focusing on cybersecurity, including a port of Zephyr OS;
+- The 2023-2024 contest, focusing on the acceleration of the MNIST digit recognition with custom extensions;
+- The 2024-2025 contest, focusing on the frequency increase (_not released yet_);
+- A treat with the support of Linux and a VGA output.
+
+[CVA6 with Xilinx Ethernet](https://github.com/cispa/CVA6-Vivado-Project-with-Xilinx-AXI-Ethernet/) is an alternative design which implements Xilinx 1G/2.5G Ethernet Subsystem on the Digilent Genesys 2 FPGA board. It has been tested with TFTP boot in u-boot and SSH in Linux.
+
+### Designs (ASIC)
+
+Here are open-source ASIC designs based on CVA6:
+
+[Polara APU](https://github.com/openhwgroup/core-v-polara-apu) is a 4-core processor made with OpenPiton, ARA and CVA6.
+
+To be completed
+
+## Business resources
+
```

Comment:
MU-Electronics is a services company having its design center in Rabat-Morocco since 2003, Specialized in ICs design from datasheet to GDSII generation, Firmware, driver & application development, Test & validation, Security implementation & support to certification.
MU-E supports your Hardware design in a flexible way, MU-E has designed IPs and Chips down to 7nm
MU-E has expertise in all the ASIC design flow (RTL Design, DFT, Verification ...)
MU-E has expertise in full custom layout, place and route.
MU-E is actively participating in European-funded projects TRISTAN and working for Thales on the verification of CVA6 IPs.

Reply:
Thanks. I have summarized a bit to approach the 60 word target.